tests/zfs-tests/tests/functional/cli_root/zpool_import/zpool_import.kshlib

```diff
@@ -383,5 +382,25 @@ function get_last_txg_synced
 	[[ $? -ne 0 ]] && return 1
 
 	echo $txg
-	return 0
+}
+
+function get_last_txg_synced_linux
+{
+	typeset pool=$1
+
+	txg=$(tail "/proc/spl/kstat/zfs/$pool/txgs" |
+	    awk '$3=="C" {print $1}' | tail -1)
+	[[ "$txg" ]] || txg=0
+	echo $txg
+}
```

Comment:
Why don't we update this Linux, FreeBSD, and illumos allow use the `zdb` approach.  No reason for this to been different if we can avoid it.